src/Symfony/Component/Ldap/Adapter/ExtLdap/Collection.php

```diff
@@ -84,7 +85,13 @@ private function initialize()
             return;
         }
 
-        $entries = ldap_get_entries($this->connection->getResource(), $this->search->getResource());
+        $con = $this->connection->getResource();
```

Comment:
What if `getResource()` returns `null` (maybe I simply miss the point where we ensure that the connection is always established before this code is executed)?